cmd/cosign/cli/copy/copy.go

```diff
@@ -102,6 +109,7 @@ func CopyCmd(ctx context.Context, regOpts options.RegistryOptions, srcImg, dstIm
 		// Copy the entity itself.
 		g.Go(func() error {
 			dst := dstRepoRef.Tag(srcDigest.Identifier())
+			dst = dst.Tag(fmt.Sprint(regOpts.RefOpts.TagPrefix, h.Algorithm, "-", h.Hex))
```

Comment:
This means we will never copy the tag itself 🤔

cc @haydentherapper

Reply:
Honestly, I'm not sure what `dstRepoRef.Tag(srcDigest.Identifier())` is even meant to do either 😅